phoenicis-scripts/src/main/java/org/phoenicis/scripts/ui/SetupUi.java

```diff
@@ -83,6 +83,12 @@
      */
     void showProgressBar(Message<ProgressControl> message, String textToShow);
 
+    /**
+     * Shows a browser inside the window and returns immediately
+     * @param textToShow The text to show to the user
+     */
+    void showBrowser(Message<BrowserControl> doneCallback, String textToShow);
```

Comment:
JavaDoc for `doneCallback` is missing

Reply:
It is missing for all the methods. Maybe we should handle this in a separate ticket